<?xml version="1.0"?>
<launch>
    <!-- Arguments -->
    <arg name="model" default="waffle_pi"/>
    <arg name="goals_folder" default="$(find robo_navi)/maps"/>

    <node pkg="robo_navi" type="move_goal.py" name = "move_goal_0">
        <param name="goals_file" value="$(arg goals_folder)/0.csv"/>
        <param name="robot_name" value="tb3_0"/>
        <param name="wait_time" value="0"/>
    </node>

    <node pkg="robo_navi" type="move_goal.py" name = "move_goal_1">
        <param name="goals_file" value="$(arg goals_folder)/1.csv"/>
        <param name="robot_name" value="tb3_1"/>
        <param name="wait_time" value="8"/>
    </node>

    <node pkg="robo_navi" type="move_goal.py" name = "move_goal_2">
        <param name="goals_file" value="$(arg goals_folder)/2.csv"/>
        <param name="robot_name" value="tb3_2"/>
        <param name="wait_time" value="16"/>
    </node>

    <node pkg="robo_navi" type="move_goal.py" name = "move_goal_3">
        <param name="goals_file" value="$(arg goals_folder)/3.csv"/>
        <param name="robot_name" value="tb3_3"/>
        <param name="wait_time" value="24"/>
    </node>

    <node pkg="robo_navi" type="move_goal.py" name = "move_goal_4">
        <param name="goals_file" value="$(arg goals_folder)/4.csv"/>
        <param name="robot_name" value="tb3_4"/>
        <param name="wait_time" value="32"/>
    </node>



    <!-- rviz -->
    <!-- <node pkg="rviz" type="rviz" name="rviz" required="true" args="-d $(find robo_navi)/rviz/multi_robo_navi.rviz"/> -->
</launch>